front/components/assistant/conversation/InputBar.tsx

```diff
@@ -387,325 +422,382 @@ export function AssistantInputBar({
                 : ""
             )}
           >
-            <div
-              className={classNames(
-                // This div is placeholder text for the contenteditable
-                contentEditableClasses,
-                "absolute -z-10 text-element-600 dark:text-element-600-dark",
-                empty ? "" : "hidden"
-              )}
-            >
-              Ask a question or get some @help
+            <div className="flex flex-row items-end pb-4">
+              <input
+                type="file"
+                ref={fileInputRef}
+                style={{ display: "none" }}
+                onChange={async (e) => {
+                  // focus on the input text after the file selection interaction is over
+                  inputRef.current?.focus();
+                  const file = e?.target?.files?.[0];
+                  if (!file) return;
+                  const res = await handleFileUploadToText(file);
+                  if (res.isErr()) {
+                    sendNotification({
+                      type: "error",
+                      title: "Error uploading file.",
+                      description: res.error.message,
+                    });
+                    return;
+                  }
+                  setContentFragmentFilename(res.value.title);
+                  setContentFragmentBody(res.value.content);
+                }}
+              />
+
+              <Tooltip
+                label="Add a document to the conversation (.txt, .pdf, .md)."
+                position="above"
+              >
+                <IconButton
+                  className="s-element-700 block"
```

Comment:
just saw you put the 's-' back? I had removed the 's-' and explained in the comments the 's-' is in sparkle/ not elsewhere. Do you disagree?

Reply:
Oh ok, it's to override the s-element-600 imposed by sparkle. IMHO a bit shaky, I would remove the class altogether and let it be s-text-element-600 from sparkle, or if important make the fix to be able to do text-element properly -- but feel free to adress how you see fit

=> however it's `text-element` not just `element` (might be a mistake from my commit suggestion if yes apologies) so this has to be changed since just 'element-700' / 's-element-700' are not real classes

Reply:
Not sure i'm being clear :sweat_smile: happy to explain first thing tom morning if needed

Reply:
Yep we should have no s- in front

Reply:
This should be part of sparkle as an option if needed

Reply:
I am removing the `s-`. I did not disagree, I just did not copy/paste your code @philipperolet.